be/src/exprs/bitmap_function.h

```diff
@@ -46,6 +46,9 @@ class BitmapFunctions {
     // the input src's ptr need to point a BitmapValue, this function will release the
     // BitmapValue memory
     static BigIntVal bitmap_finalize(FunctionContext* ctx, const StringVal& src);
+    // Get the bitmap cardinality, the difference from bitmap_finalize method is
+    // bitmap_get_value method doesn't free memory
+    static BigIntVal bitmap_get_value(FunctionContext* ctx, const StringVal& src);
```

Comment:
How about `bitmap_get_cardinality` ?

Reply:
If fact, `bitmap_finalize` and `bitmap_get_value` both compute bitmap cardinality, like `hll_finalize` and `hll_get_value`.  Use `get_value` as function name, because I want the fucntion name keep consistent with `_get_value_fn` in BE `AggFnEvaluator` and `getValueFnSymbol` in FE `AggregateFunction`.

I will update the comment to emphasize this point.

Reply:
OK